src/node_worker.cc

```diff
@@ -661,6 +663,7 @@ void Worker::StartThread(const FunctionCallbackInfo<Value>& args) {
   ASSIGN_OR_RETURN_UNWRAP(&w, args.This());
   Mutex::ScopedLock lock(w->mutex_);
 
+  workers[w->thread_id_.id] = w;
```

Comment:
Can you add a comment to why we are storing the reference? It might be useful for other contributors

Reply:
Done